app/views/idv/in_person/address.html.erb

```diff
@@ -6,35 +6,51 @@
   <%= render PageHeadingComponent.new.with_content(t('in_person_proofing.headings.address')) %>
 <% end %>
 
-<p>
-  <%= t('in_person_proofing.body.address.info') %>
-  <%= new_window_link_to(
-        t('in_person_proofing.body.address.learn_more'),
-        MarketingSite.help_center_article_url(
-          category: 'verify-your-identity',
-          article: 'verify-your-identity-in-person',
-        ),
-      ) %>
-</p>
+<% unless IdentityConfig.store.in_person_capture_secondary_id_enabled %>
+  <p>
+    <%= t('in_person_proofing.body.address.info') %>
+    <%= new_window_link_to(
+          t('in_person_proofing.body.address.learn_more'),
+          MarketingSite.help_center_article_url(
+            category: 'verify-your-identity',
+            article: 'verify-your-identity-in-person',
+          ),
+        ) %>
+  </p>
+<% end %>
 
 <%= simple_form_for(
       form, url: url_for, method: 'PUT',
             html: { autocomplete: 'off' }
     ) do |f| %>
+    <% if IdentityConfig.store.in_person_capture_secondary_id_enabled %>
+      <%= render ValidatedFieldComponent.new(
+            collection: us_states_territories,
+            form: f,
+            label: t('idv.form.state'),
+            label_html: { class: 'usa-label' },
+            name: :state,
+            prompt: t('in_person_proofing.form.address.state_prompt'),
+            required: true,
+            selected: pii[:state],
+          ) %>
+    <% end %>
   <%= render ValidatedFieldComponent.new(
         form: f,
         input_html: { value: pii[:address1] },
         label: t('idv.form.address1'),
         label_html: { class: 'usa-label' },
+        hint: '',
         maxlength: 255,
         name: :address1,
         required: true,
       ) %>
   <%= render ValidatedFieldComponent.new(
         form: f,
         input_html: { value: pii[:address2] },
-        label: t('idv.form.address2_optional'),
+        label: IdentityConfig.store.in_person_capture_secondary_id_enabled ? t('idv.form.address2') : t('idv.form.address2_optional'),
```

Comment:
Could this be provided by a presenter instead?

Reply:
After reviewing the current presenter usages, it looks like switching to a presenter may be more trouble than it's worth here.